src/frontend/src/lib/derived/network-tokens.derived.ts

```diff
@@ -10,7 +10,7 @@ import { derived, type Readable } from 'svelte/store';
 /**
  * All user-enabled tokens matching the selected network or chain fusion.
  */
-const enabledNetworkTokens: Readable<Token[]> = derived(
+export const enabledNetworkTokens: Readable<Token[]> = derived(
```

Comment:
Since we expose it, do you want to create a test for it? In another PR.

Reply:
so, I wanted to do it in a separate PR, because it requires to mock `page` store and I found out it is not that easy

or maybe you already encountered something similar? any suggestion?

Reply:
I think we are on the same page, I also wrote "In another PR" 😉.
Coolio.

Reply:
ah! i didn't read it, sorry eheh